content/pages/faq.md

```diff
@@ -34,6 +34,13 @@ Our women-only events are not publicly advertised and we always specify
 Please see the **Volunteer Opportunities** section on our [About
 page]({filename}/pages/about.md)!
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources that are available for you to read
+during office hours. They include:
+* Quarterly publications from Model View Culture 
```

Comment:
oops - didn't bullet point properly. Pretty sure you want an extra line before this one.

you can see here: https://wics-site-pr-217.herokuapp.com/faq/